packages/core/src/lib/TwaManifest.ts

```diff
@@ -70,6 +70,7 @@ export type FallbackType = 'customtabs' | 'webview';
  * navigationColor: '<%= themeColor %>', // The color used for the navigation bar.
  * backgroundColor: '<%= backgroundColor %>', // The color used for the splash screen background.
  * enableNotifications: false, // Set to true to enable notification delegation.
+ * enableSiteSettingsShortcut: false, // Set to true to enable a dynamic shortcut into site settings.
```

Comment:
Maybe this should be `: true` instead of `: false` because it defaults to true? (I'm not quite sure what the structure of the comment here is though.)